README.md

```diff
@@ -25,20 +25,20 @@ inference.
 
 2. Install prerequisites by running the following:
     ```
-    sudo apt-get install python3-pip python3-venv virtualenv
+    sudo apt-get install python3-pip python3-venv
     ```
 
 3. Create and activate virtual environment:
     ```
-    virtualenv venv
+    python3 -m venv venv
     source venv/bin/activate
     ```
 
 4. Install `ote_cli` package:
     ```
     pip3 install -e ote_cli/
     ```
-    
+
 5. Instantiate templates and create virtual environments:
    ```
    python3 tools/instantiate.py --destination model_templates --verbose --init-venv
```

Comment:
Kinda off-topic, but is it still relevant?

Reply:
Will fix in next pr